management/database/client.go

```diff
@@ -0,0 +1,154 @@
+package database
```

Comment:
can we call this `sql` to be consistent with the nuget naming scheme as well https://www.nuget.org/packages/Microsoft.WindowsAzure.Management.Sql/ ? We have a product called DocumentDB as well and that's `database`, too.